elements/pfe-sass/maps/_typography.scss

```diff
@@ -81,3 +81,148 @@ $pf-type-sizing--titles: (
+    sm: (
+      FontSize: pfe-var(content--small--FontSize)
+    )
+  ),
+  /// The variable stack includes a hook just for headings when they appear in the 
+  /// light DOM within components: --pf-c--FontSize
+  ///
+  /// Components can't control the font size of block level elements like HTML headings 
+  /// when they are in the light DOM.  A component _could_ redefine the variables for 
+  /// standard headings by using the variable hooks such as: `--pf-c--content--h4--FontSize` 
+  /// but it can't know if the light DOM might contain an H1, H2, H3, etc. it shouldn't
+  /// redefine the variables for all possible headings. Instead, there is one dedicated
+  /// variable for this purpose. 
+  content: (
+    h1: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   var(--pf-c--FontSize, pfe-var(content--h1--FontSize)),
+      LineHeight: map-get($pfe-typography-base, line-height--heading),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm), // 16px
+    ),
+    h2: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   var(--pf-c--FontSize, pfe-var(content--h2--FontSize)),
+      LineHeight: map-get($pfe-typography-base, line-height--heading),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm), // 16px
+    ),
+    h3: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   var(--pf-c--FontSize, pfe-var(content--h3--FontSize)),
+      LineHeight: map-get($pfe-typography-base, line-height--heading),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm), // 16px
+    ),
+    h4: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   var(--pf-c--FontSize, pfe-var(content--h4--FontSize)),
+      LineHeight: map-get($pfe-typography-base, line-height--heading),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm), // 16px
+    ),
+    h5: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   var(--pf-c--FontSize, pfe-var(content--h5--FontSize)),
+      LineHeight: map-get($pfe-typography-base, line-height--heading),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm), // 16px
+    ),
+    h6: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   var(--pf-c--FontSize, pfe-var(content--h6--FontSize)),
+      LineHeight: map-get($pfe-typography-base, line-height--heading),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm), // 16px
+    ),
+  )
```

Comment:
Overall I think this is cool, I'm really wanting to know what the output is though. Like will this print , in this order, these variables in the stack?
 - component `--pf-c--FontSize`
 - content block h6 `--pf-c-content--h6--FontSize `
 - global t-shirt size  `--pf-global--FontSize--sm`
 - fallback value  `0.875rem;`

Reply:
The mixin really handles a lot about the stack, my goal was for the map to contain the data points and the mixin to assemble how it's being output and which properties to use.